decidim-courses/app/presenters/decidim/courses/course_presenter.rb

```diff
@@ -12,18 +12,12 @@ class CoursePresenter < SimpleDelegator
       def hero_image_url
         return if course.hero_image.file.blank?
 
-        uri = URI(course.hero_image.file.file)
-        return uri unless uri.scheme.nil?
-
```

Comment:
why remove this?

Reply:
Because it is not needed, in Decidim they only have 
`URI.join(decidim.root_url(host: course.organization.host), course.hero_image_url).to_s`
and it works properly, we can leave this lines but they're redundant.